frontend/src/components/LoginAndRegistrationPage/validators.tsx

```diff
@@ -1,49 +1,40 @@
 import {REQUIRED} from "../../utils/constants";
 
 export const validatePassword = (values: string) => {
-    let error = "";
     const passwordRegex = /^(?=.*[0-9])(?=.*[a-z])(?=.*[A-Z]).{1,}$/;
     if (!values) {
-        error = REQUIRED;
+        return REQUIRED;
     } else if (!passwordRegex.test(values)) {
-        error = "Hasło musi zawierać przynajmniej jedną cyfrę i co najmniej jedną małą i jedną wielką literę";
+        return  "Hasło musi zawierać przynajmniej jedną cyfrę i co najmniej jedną małą i jedną wielką literę";
     }
-    return error;
 };
 
 export const validateConfirmPassword = (pass: any, value: any) => {
-    let error = "";
     if (!value) {
-        error = REQUIRED;
+        return REQUIRED;
     } else if (pass && value) {
         if (pass !== value) {
-            error = "Hasła się różnią.";
+            return "Hasła się różnią.";
         }
     }
-    return error;
 };
 
 export const validateEmail = (email: string) => {
     const regexEmail = /^(?=.{1,64}@)[A-Za-z0-9_-]+(\\.[A-Za-z0-9_-]+)*@[^-][A-Za-z0-9-]+(\.[A-Za-z0-9-]+)*(\.[A-Za-z]{2,})$/;
-    let error = "";
 
     if(!email){
-        error = REQUIRED;
+        return REQUIRED;
     } else if (!regexEmail.test(email)){
-        error = "Podaj poprawny adres email.";
+        return  "Podaj poprawny adres email.";
```

Comment:
Nie wiem czemu, ale nie mogę podać takiego maila, na backendzie rejestracja za pomocą takiego adresu działa, tutaj dostaję błąd
![image](https://user-images.githubusercontent.com/37903018/171993218-14005db6-3928-43fe-b0dd-02641400561d.png)

Reply:
Korzystam z regexa z twojego backendu więc odpowiedz sobie sam ...

Reply:
A tak poważnie to jestem w trakcie szukania przyczyny takiego zachowania 😅

Reply:
Zmieniłem całego regexa dla maila. Daj znać czy ten nowy spełnia wszystkie założenia

Reply:
Wg dokumentacji co mamy na backendzie, do takiej sytuacji nie powinno dojść, że mamy `-` zaraz po `@`
![image](https://user-images.githubusercontent.com/37903018/171994285-171e453f-8e87-4e8f-b3fc-1c8a0d730014.png)

A tutaj powinno na spokojnie przejść, gdzie `-` mamy w pierwszej części
![image](https://user-images.githubusercontent.com/37903018/171994309-f2a90959-215b-4657-8a82-67e9540b0a53.png)